libs/wire-api/src/Wire/API/User/Activation.hs

```diff
@@ -117,54 +146,24 @@ data Activate = Activate
   }
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform Activate)
+  deriving (ToJSON, FromJSON, S.ToSchema) via Schema Activate
 
-modelActivate :: Doc.Model
-modelActivate = Doc.defineModel "Activate" $ do
-  Doc.description "Data for an activation request."
-  Doc.property "key" Doc.string' $ do
-    Doc.description "An opaque key to activate, as it was sent by the API."
-    Doc.optional
-  Doc.property "email" Doc.string' $ do
-    Doc.description "A known email address to activate."
-    Doc.optional
-  Doc.property "phone" Doc.string' $ do
-    Doc.description "A known phone number to activate."
-    Doc.optional
-  Doc.property "code" Doc.string' $
-    Doc.description "The activation code."
-  Doc.property "label" Doc.string' $ do
```

Comment:
does was a lie before, right?  the json instances didn't know about this field?

Reply:
Yeah, I noticed that, too.